composer.json

```
@@ -28,12 +28,12 @@
     "require": {
         "php": "^7.3",
         "laravel/nova": "^2.0|^3.0",
```

```suggestion
        "laravel/nova": "^3.0",
```

Is Nova 2 supported by Laravel 7? If not, we might as well drop support for it too.

---

Also correct, have dropped it :)